### PR TITLE
Add Hive 3 metastore config for CSV support

### PR DESCRIPTION
**Pull request by mosabua**
Aimed to fix https://github.com/prestosql/presto/issues/2678

To be honest I am not sure if this is the correct place or if we need to create a separate section about the Hive metastore configuration beyond the properties and add it there. Or link both sections.

Also is the content I wrote correct or am I missing a bunch of things?

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -40,6 +40,18 @@ The following file types are supported for the Hive connector:
 * CSV (using ``org.apache.hadoop.hive.serde2.OpenCSVSerde``)
 * TextFile
 
+In order to enable first-class support for CSV files, you need to add the
+following property definition to the Hive metastore configuration file
```
Comment (findepi) [inline presto-docs/src/main/sphinx/connector/hive.rst]: this is needed for Hive 3 only.

-- this is the only place where we set this in our tests 
https://github.com/prestosql/docker-images/blob/4aadfb19cf5a3d7a2576f4cd670c9ced21debf53/prestodev/hdp3.1-hive/files/etc/hive/conf/hive-site.xml#L69-L70

Comment (findepi) [reply presto-docs/src/main/sphinx/connector/hive.rst]: @electrum, maybe we _also_ could catch exception somewhere around https://github.com/prestosql/presto/blob/ebe16716fb76e4c4883cf7196d9b1087edb51582/presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java#L89 and provide more actionable message directly to the user?

**Comment by mosabua** [comment]
Ready again @findepi

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -40,6 +40,18 @@ The following file types are supported for the Hive connector:
 * CSV (using ``org.apache.hadoop.hive.serde2.OpenCSVSerde``)
 * TextFile
 
+In order to enable first-class support for CSV files when using Hive 3.x, you
```
Comment (findepi) [inline presto-docs/src/main/sphinx/connector/hive.rst]: I missed that earlier; same applies to Avro tables with `avro_schema_url`.

For ref, we're talking about:
https://github.com/prestosql/presto/blob/ebe16716fb76e4c4883cf7196d9b1087edb51582/presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/BridgingHiveMetastore.java#L88-L89

Comment (mosabua) [reply presto-docs/src/main/sphinx/connector/hive.rst]: Updated and ready to merge again.

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**